code/modules/materials/materials.dm

```diff
@@ -187,7 +187,7 @@ var/list/name_to_material
 	if(islist(composite_material))
 		for(var/material_string in composite_material)
 			temp_matter[material_string] = composite_material[material_string]
-	else if(SHEET_MATERIAL_AMOUNT)
```

Comment:
this always returned true before, but it seems like the author may have intended something else

Have a look at surrounding code and see if this was a bug